app/src/main/java/com/lastaoutdoor/lasta/ui/theme/Type.kt

```diff
@@ -9,10 +9,8 @@ import androidx.compose.ui.unit.sp
 import com.lastaoutdoor.lasta.R
 
 // import montserrat font
-val customFontFamily =
-    FontFamily(
-        Font(R.font.montserratreg) // Replace "your_custom_font" with the name of your font file
-        )
+val customFontFamily = FontFamily(Font(R.font.montserratreg))
+val customFontFamilyItalic = FontFamily(Font(R.font.montserratitalic))
```

Comment:
Nice addition, I guess we can maybe try using some other places in the app

Reply:
+1